src/frontend/apps/drive/package.json

```diff
@@ -10,7 +10,7 @@
     "build-theme": "cunningham -g css,scss -o src/styles && mv src/styles/cunningham-tokens.scss src/styles/cunningham-tokens-sass.scss"
   },
   "dependencies": {
-    "@gouvfr-lasuite/ui-kit": "0.4.0",
+    "@gouvfr-lasuite/ui-kit": "0.4.1",
     "@openfun/cunningham-react": "3.0.0",
```

Comment:
Ditto ability to create folder

<img width="824" alt="Capture d’écran 2025-04-25 à 14 24 35" src="https://github.com/user-attachments/assets/2824b2f8-d605-40d4-903a-7a4673da0a73" />